tolicodes.com-frontend/src/components/App/index.tsx

```diff
@@ -1,42 +1,30 @@
-import React from 'react';
-import { GeistProvider, CssBaseline } from '@geist-ui/react'
+import React from "react";
 
-import { Container } from './Container';
+import { GSBase } from "../../constants/styles/global-styles";
 
-import { Header } from '../Header';
-import { SiteName } from '../SiteName';
-import { SocialNetworkMenu } from '../SocialNetworkMenu';
-import { Menu } from '../Menu';
-import { YouTubeVideo } from '../YouTubeVideo';
-import { Description } from '../Description';
+import { Container } from "./Container";
 
-import { data } from '../../data/data'
+import { Header } from "../Header";
+import { YouTubeVideo } from "../YouTubeVideo";
+import { Description } from "../Description";
+
+import { data } from "../../data/data";
 
 function App() {
   return (
     <Container>
-      <GeistProvider>
-        <CssBaseline>
-          <Header>
-            <SiteName 
-              siteLogoSrc={data.siteName.siteLogoSrc} 
-              siteTagline={data.siteName.sitTagLine} 
-            />
-            <SocialNetworkMenu links={data.socialMediaLinks}/>
-            <Menu links={data.menuLinks} />
-          </Header>
-          <YouTubeVideo
-            src={data.youTubeVideo.src}
-            height={data.youTubeVideo.height}
-            title={data.youTubeVideo.title}
-          />
-          <Description title={data.description.title}>
-            {data.description.aboutMeText}
-          </Description>
-        </CssBaseline>
-      </GeistProvider>
+      <Header />
+      <YouTubeVideo
+        src={data.youTubeVideo.src}
```

Comment:
Destructure

Reply:
This is not my code and I didn't start working on this part just yet, so ignore for now.